.env.example

```diff
@@ -76,15 +79,18 @@ BIT_API_URLS_MAINNET_BALANCE_CHECK=#endpoint for checking balance against a supp
 BIT_API_URLS_MAINNET_UNSPENT_OUTPUTS=#endpoint for fetching utxos against a supplied mainnet address
 BIT_API_URLS_MAINNET_BROADCAST=#endpoint for broadcasting a mainnet txnHex
 BIT_API_URLS_MAINNET_TX_DECODE=#endpoint for decoding a mainnet txnHex
-BIT_API_URLS_MAINNET_TX_FETCH_URL=#endpoint for fetching transactions against a supplied mainnet address 
+BIT_API_URLS_MAINNET_TX_FETCH_URL=#endpoint for fetching transactions against a supplied mainnet address
 BIT_API_URLS_MAINNET_TX_LIMIT=#mainnet txn fetch limit specifier (ex: ?limit=100)
 
 # CIPHER_SPEC
 BIT_CIPHER_ALGORITHM=#en/de-cryption algorithm
 BIT_CIPHER_SALT=#en/de-cryption salt
 BIT_CIPHER_KEYLENGTH=#key length for the cipher
 
+
 # SSS Variables
+BIT_KEY_STRETCH_ITERATIONS=# (default: 10000)
```

Comment:
I based this on the [latest NIST guidelines](https://pages.nist.gov/800-63-3/sp800-63b.html#sec5) for PBKDF2 iteration count